### PR TITLE
US276527: bug fixing related to gitlab

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements
US276527: bug fixing related to gitlab

### CODE REVIEW

File: jenkins-build-pack-api/Jenkinsfile [hunk 1]
```diff
@@ -78,7 +78,7 @@ node() {
 	
     stage('Checkout code base') {
 		if (params.domain  && params.domain.trim() != "" ) {			
-			if(domain != "platform"){
+			if(domain != "platform" && configLoader.SCM.TYPE != "gitlab"){
```
Comment (suryajak) [inline jenkins-build-pack-api/Jenkinsfile]: Why is this required? We ideally shouldn't have the diff between bitbucket and gitlab, so please comment above this line why is that required?

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 2]
```diff
@@ -73,7 +73,7 @@ node ()  {
 	def domain = params.domain
 
 	if(params.domain && params.domain.trim() != ""){
-		if(params.domain != "platform"){
+		if(params.domain != "platform" && configLoader.SCM.TYPE != "gitlab" ){
```
Comment (suryajak) [inline jenkins-build-pack-lambda/Jenkinsfile]: Same as above - Why is this required?

File: jenkins-build-pack-website/Jenkinsfile [hunk 3]
```diff
@@ -78,7 +78,9 @@ node ()  {
     stage('Checkout code base') {
 
 		if (domain && domain.trim()!="") {
-			repo_name = params.domain + "-" + service
+			if(configLoader.SCM.TYPE != "gitlab" ){
```
Comment (suryajak) [inline jenkins-build-pack-website/Jenkinsfile]: Same as above - Why is this required?

File: jenkins-build-pack-api/Jenkinsfile (hunk 1 above)
Comment (SiniWilson) [reply jenkins-build-pack-api/Jenkinsfile]: This is because the diversity in the repo naming of platform services and user services.
In Gitlab the same webhook trigger job is using for platform services and user services.
So this if block is needed.

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 2 above)
Comment (SiniWilson) [reply jenkins-build-pack-lambda/Jenkinsfile]: This is because the diversity in the repo naming of platform services and user services.
In Gitlab the same webhook trigger job is using for platform services and user services.
So this if block is needed.

File: jenkins-build-pack-website/Jenkinsfile (hunk 3 above)
Comment (SiniWilson) [reply jenkins-build-pack-website/Jenkinsfile]: This is because the diversity in the repo naming of platform services and user services.
In Gitlab the same webhook trigger job is using for platform services and user services.
So this if block is needed.

File: jenkins-build-pack-api/Jenkinsfile (hunk 1 above)
Comment (suryajak) [reply jenkins-build-pack-api/Jenkinsfile]: I am not sure I quite follow. I believe this is a bug that needs to be fixed elsewhere (the deviation between how gitlabs jobs are triggered vs bitbucket jobs are triggered). Can you point those differences?
